src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -354,6 +355,13 @@ export default {
 			type: String,
 			default: 'bottom',
 		},
+		/**
+		 * Entry aria details
+		 */
+		ariaDescription: {
+			type: String,
+			default: '',
```

Comment:
Should be null by default :)
![image](https://user-images.githubusercontent.com/14975046/158443998-f8737529-4afe-4a48-a78f-505235e40ea7.png)